dv/views/api.py

```diff
@@ -144,20 +144,29 @@ def overview(request):
             bilateral_initiatives[(FM_NORWAY, bi.state_id)].append(bi.code)
 
     RELEVANT_INDICATORS = {
-        'Number of people engaged in civil society organisation activities': 'people_civil_society',
-        'Estimated annual CO2 emissions reductions': 'co2_emissions_reduction',
-        'Number of researchers supported': 'supported_researchers',
-        'Number of professional staff trained': 'staff_trained',
-        'Number of jobs created': 'jobs_created',
+        # FUNDING_PERIOD_ID
+        2: {
+            'Estimated CO2 reduction and/or avoidance in tonnes/year': 'annual_c02_emissions_reduced',
+            'Number of green jobs created': 'green_jobs_created',
+            'Number of NGOs/small organisations reporting strengthened capacity': 'ngos_small_organisations_supported',
+            'Number of beneficiaries reporting improved access to basic and welfare services': 'people_improved_access'
+        },
+        3: {
+            'Number of people engaged in civil society organisation activities': 'people_civil_society',
+            'Estimated annual CO2 emissions reductions': 'co2_emissions_reduction',
+            'Number of researchers supported': 'supported_researchers',
+            'Number of professional staff trained': 'staff_trained',
+            'Number of jobs created': 'jobs_created',
+        }
```

Comment:
Let's rename this dictionary to `OVERVIEW_INDICATORS`, move it to `dv/lib/utils.py`, import it here, and do something like:
```relevant_indicators = OVERVIEW_INDICATORS[period_id]```
only once.